OperacoesListaSequencial.cpp

```diff
@@ -0,0 +1,459 @@
+#include <stdlib.h>
+#include <iostream>
+#include <new>
+#include <string>
+
+using namespace std;  // para utilizar cout
+
+// Struct para pessoa
+struct pessoa {
+  string nome;
+  int rg;
+};
```

Comment:
1. this should be a generalistic struct, not a struct for 'pessoa';
2. shouldn't a linked list contain a pointer to itself and a pointer to the next on the list?

Reply:
study this: https://www.bogotobogo.com/cplusplus/linkedlist.php

Reply:
A library that has been inserted is for sequential lists only. I will attach to linked lists after the changes.